src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,119 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_BASE_SIZE = 16;
+    private static final float RESIZE_FACTOR = 0.75f;
+    private static final int RESIZE_COEFFICIENT = 2;
+
+    private Object[] table;
+    private int size;
```

Comment:
Remember about access modifiers. The field `size` should be private to adhere to encapsulation principles.